.github/workflows/e2e.yml

```diff
@@ -53,3 +53,7 @@ jobs:
           DATABASE_URL: "postgres://postgres@localhost:5432/postgres"
         working-directory: ./e2e/packages/sync-test
         run: pnpm test:ci
+
+      - name: Run tests
+        working-directory: ./e2e
+        run: pnpm run test:ci
```

Comment:
we should combine this with the above so we don't run sync tests twice

Reply:
done